containers/reverse-proxy/envoy-start

```diff
@@ -19,12 +19,29 @@ else
   exit 1
 fi
 
-echo "$(date -Is) Changing the port values in etc/envoy/envoy.yaml to ${SERVICE_PORT} and ${PORT0}"
+if [ ${HEALTH_CHECK_PORT_INDEX:-0} -gt 0 ]; then
+    echo "$(date -Is) HEALTH_CHECK_PORT_INDEX is positive"
+    PROXY_PORT_COUNT=2
+else
+    PROXY_PORT_COUNT=1
+fi
+
+CONFIG_YAML=/etc/envoy/envoy${PROXY_PORT_COUNT}.yaml
```

Comment:
As mentioned above, I'd prefer to use more descriptive names for the envoy YAML files.